distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-1"]
+= Release notes for {DTProductName} 3.1
+:context: distributed-tracing-rn-3-1
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+[id="distributed-tracing-rn_3-1_component-versions"]
```

Comment:
I don't usually see underscores used in IDs unless it's a specific case (additional resources, prerequisites) so per the guidelines here I'd recommend only using dashes for these https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-assembly-files

e.g. `[id="distributed-tracing-rn-3-1-component-versions"]`

Reply:
Thank you. I'll add these fixes to the next release's epic.